docs/core/linux-prerequisites.md

```diff
@@ -88,145 +105,184 @@ The installer bash script is used in automation scenarios and non-admin installa
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-## Install .NET Core dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
+## Install .NET Core 1.x dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
+
+To install **.NET Core 1.x** dependencies on RHEL Server:
+
+1. Verify that all previous .NET Core preview versions have been removed.
 
 > [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+> Before you start, remove any **previous preview** versions of .NET Core from your system.
 
-### Verify and Enable the .NET channel for RHEL 7 Server
-To install .NET Core dependencies on RHEL Server:
+2. Enable the Red Hat .NET channel, available under your RHEL 7 Server subscription.
 
-1. Enable the Red Hat .NET channel, available under your RHEL 7 Server subscription. 
     * For Red Hat Enterprise 7 Server, use:
+    
          ```bash
-        subscription-manager repos --enable=rhel-7-server-dotnet-rpms
-        ```
+         subscription-manager repos --enable=rhel-7-server-dotnet-rpms
+         ```
+    
     * For Red Hat Enterprise 7 Workstation, use:
-         ```bash
+    
+        ```bash
         subscription-manager repos --enable=rhel-7-workstation-dotnet-rpms
-        ```
+         ```
+    
     * For Red Hat Enterprise 7 HPC Compute Node, use:
-         ```bash
+    
+        ```bash
         subscription-manager repos --enable=rhel-7-hpc-node-dotnet-rpms
         ```
 
-2. Install the scl tool.
+3. Install the scl tool.
+
     ```bash
     yum install scl-utils
     ```
-3. Install .NET Core 1.1 (and all dependencies):
+4. Install .NET Core 1.1 (and all dependencies):
     ```bash
     yum install rh-dotnetcore11
     scl enable rh-dotnetcore11 bash
     ```
-4. Run the `dotnet --help` command to prove the installation succeeded.
+5. Run the `dotnet --help` command to prove the installation succeeded.
 
      ```bash
      dotnet --help
      ```
-> [!NOTE]
-> For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
 
+For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
+
+## Install .NET Core for Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10 & Linux Mint 17, Linux Mint 18 (64 bit)
 
-## Install .NET Core for Ubuntu 14.04, 16.04, 16.10 & Linux Mint 17, 18 (64 bit)
+1. Verify that all previous .NET Core preview versions have been removed.
 
 > [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+> Before you start, remove any **previous preview** versions of .NET Core from your system.
 
 ### Add the dotnet apt-get feed
```

Comment:
why is this a subsection? seems weird to have that in the middle of the steps. Can it be removed?